brave/ui/app/pages/connect-accounts/connect-accounts.component.js

```diff
@@ -0,0 +1,175 @@
+import CloseIcon from '../first-time-flow/welcome-modal/assets/close-icon'
+import CloseIconDark from '../first-time-flow/welcome-modal/assets/close-icon-dark'
+import {
+  BRAVE_CONNECT_WALLETS_ROUTE,
+  INITIALIZE_CREATE_PASSWORD_ROUTE,
+  BRAVE_BITGO_WALLET_INDEX
+} from '../../helpers/constants/routes'
+const { supportedCoins } = require('../../../../app/scripts/controllers/bitgo')
+
+module.exports = class BraveConnectAccounts extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    history: PropTypes.object,
+    createBitGoWallet: PropTypes.func,
+    bitGoCreatedWallets: PropTypes.array,
+  }
+
+  constructor (props) {
+    super(props)
+
+    const checkedAssets = {}
+    const { bitGoCreatedWallets } = this.props
+    this.bitGoCreateableAssets = {}
+
+    for (let coin in supportedCoins) {
+      if (!bitGoCreatedWallets.includes(coin)) {
+        this.bitGoCreateableAssets[coin] = supportedCoins[coin]
+      }
+    }
+
+    Object.keys(this.bitGoCreateableAssets).map((key) => {
+      checkedAssets[key] = false
+    })
+
+    this.state = {
+      checkedAssets,
+      showBitGoModal: false
+    }
+  }
+
+  onCreate = () => {
+    this.props.history.push(INITIALIZE_CREATE_PASSWORD_ROUTE)
+  }
+
+  onConnectFinish = () => {
+    this.props.history.push(BRAVE_CONNECT_WALLETS_ROUTE)
+  }
+
+  onCreateBitGoWallets = () => {
+    const { checkedAssets } = this.state
+
+    for (let coin in checkedAssets) {
+      if (checkedAssets[coin]) {
+        this.props.createBitGoWallet(coin)
+      }
+    }
+
+    this.props.history.push(BRAVE_BITGO_WALLET_INDEX)
+  }
+
+  onConnectBitGoWallets = () => {
+    this.setState({ showBitGoModal: !this.state.showBitGoModal })
+  }
+
+  checkAsset = (asset) => {
+    const { checkedAssets } = this.state
+    checkedAssets[asset] = !checkedAssets[asset]
+    this.setState({ checkedAssets })
+  }
+
+  getCryptoImage = (asset) => {
+    // temporary
+    if (asset === 'bsv' || asset === 'btg' || asset === 'eos' || asset === 'algo') {
+      asset = 'btc'
+    }
+
+    return (
+      <img src={`images/${asset}-small.png`} />
+    )
+  }
+
+  renderBitGoModal () {
+    const { checkedAssets } = this.state
+
+    return (
+      <div className="welcome-modal">
+        <div className="__modal">
+          <div className="__close" onClick={this.onConnectBitGoWallets}>
+            <div className="close-light">
+              <CloseIcon />
+            </div>
+            <div className="close-dark">
+              <CloseIconDark />
+            </div>
+          </div>
+          <div className="__content __bitgo" style={{ textAlign: 'left' }}>
+            <BitGoLogoIcon />
+            <h3>
+              {'Select all crypto assets you would like to interact with.'}
+            </h3>
+            <p>{'A new wallet will be generated for each coin checked. Two of three private keys are stored on BitGo. The third key will be stored securely on Brave.'}</p>
```

Comment:
this message should be inverted. (2/3 keys are stored on brave, 1 is stored on bitgo)

may be worth mentioning that any transaction requires 2/3 keys

Reply:
also this should maybe link to https://www.bitgo.com/terms

Reply:
Got it, I'll fix this when I do localization